src/reducers/reducerUtil.js

```diff
@@ -44,6 +44,19 @@ function handlePagedAction(action, state) {
                 cursor: cursor
             });
         }
+        case ActionTypes.SINGLE: {
+            let _contents = state.items;
+            _contents = {
+                ..._contents,
```

Comment:
I honestly don't know for certain, i saw it that way in examples and assumed is was the common syntax... it's been in reducer util the whole time for all the paged reducers.